Lib/json/decoder.py

```diff
@@ -351,13 +351,18 @@ def raw_decode(self, s, idx=0):
         """Decode a JSON document from ``s`` (a ``str`` beginning with
         a JSON document) and return a 2-tuple of the Python
         representation and the index in ``s`` where the document ended.
+        Whitespace at the beginning of the document will be ignored.
+
+        Optionally, ``idx`` can be used to specify an offset in ``s``
+        where the document begins.
 
         This can be used to decode a JSON document from a string that may
         have extraneous data at the end.
 
         """
+        _w = WHITESPACE.match
```

Comment:
Should we pass this as a private argument of raw_decode? That is more consistent with the other methods in this file